src/request/components/request-detail-panel-logging.jsx

```diff
@@ -45,6 +46,30 @@ function getRowClass(message) {
     return rowClass;
 }
 
+function getDisplayText(level) {
+    return _.startCase(level);
+}
+
+function getIconName(level) {
```

Comment:
Originally the intent was to use the class defined in `getRowClass` to load in the corresponding svg icon. I think this would still be my preferred approach than loading in a whole new font. But I'm happy to pull this in in the short term.